.github/workflows/wasm.yml

```diff
@@ -0,0 +1,62 @@
+name: "rebuild wasm"
+
+on:
+  workflow_dispatch:
```

Comment:
As per @bartlomieju we can change the trigger to be on PR instead of workflow dispatch, with an if statement in the job to only run when the `hash/` path is modified, I think we have something solid here.